src/sentry/tasks/organizationmapping.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from datetime import timedelta
+
+from django.db import connection
+from django.utils import timezone
+
+from sentry.models.organizationmapping import OrganizationMapping
+from sentry.services.hybrid_cloud.organization import organization_service
+from sentry.tasks.base import instrumented_task
+from sentry.utils import metrics
+
+ORGANIZATION_MAPPING_EXPIRY = timedelta(hours=4)
+
+logger = logging.getLogger("tasks.releasemonitor")
+
+
+@instrumented_task(
+    name="sentry.hybrid_cloud.tasks.organizationmapping",
+    queue="#####TODO",
+    default_retry_delay=5,
+    max_retries=5,
+)  # type: ignore
+def organizationmapping_repair(**kwargs) -> None:
+    metrics.incr("sentry.hybrid_cloud.tasks.organizationmapping.start", sample_rate=1.0)
+    with metrics.timer("sentry.hybrid_cloud.tasks.organizationmapping.repair", sample_rate=1.0):
+        expiration_threshold_time = timezone.now() - ORGANIZATION_MAPPING_EXPIRY
+        # Enumerate unverified mappings, mark verified if they exist in the region silo, delete them if they're > 4 hours old
+        for mapping in OrganizationMapping.objects.filter(verified=False).iterator():
```

Comment:
`DISABLE_SERVER_SIDE_CURSORS` is set to `True` in prod, so this won't use server side cursors https://docs.djangoproject.com/en/2.2/ref/models/querysets/#with-server-side-cursors

If this is likely to be a lot of rows, use `RangeQuerySetWrapper`